PelotonEppSdkTests/CreditCardTransactionRequestTests.cs

```diff
@@ -51,6 +51,42 @@ public void TestCreditCardTokenTransaction()
         }
 
         [TestMethod]
+        public void TestCreditCardTokenTransactionMaxOrderNumberLength()
```

Comment:
These tests were done a while ago, but never made it into the branch. Going add them in this omnibus commit.